source/triggers.txt

```diff
@@ -125,3 +128,25 @@ may not exceed the number of available change streams.
 
 :ref:`Learn more about the number of supported change streams for Atlas tiers.
 <mongodb-service-limitations-change-streams>`
+
+.. _trigger-diagnose-duplicate-events:
+
+Diagnose Duplicate Events
+-------------------------
+
+During normal Trigger operation, Triggers do not send duplicate events.
+However, when some failure or error conditions occur, Triggers may deliver 
+duplicate events. You may see a duplicate Trigger event when:
+
+- A server responsible for processing and tracking events experiences a 
+  failure. This failure prevents the server from recording its progress in a 
+  durable or long-term storage system, making it "forget" it has processed 
+  some of the latest events.
+- Using unordered processing where events 1-N are sent simultaneously. If 
```

Comment:
```suggestion
- Using unordered processing where events 1 through N are sent simultaneously. If 
```
I read as "1 minus N" because I wasn't expecting it. The phrase could also be deleted, and it wouldn't make much of a difference, considering the variable N isn't used any where else.

Reply:
Gotcha. The original example was "1-10" but I changed it to "N" because I didn't want to imply that 10 was some sort of limit on unordered processing. With this feedback, I see that the more concrete example is helpful, and I do think having the numbers explicit here clarifies the example, so I'll change it back to 10. 👍